### ISSUE TITLE
Final Exam

### ISSUE THREAD

**Issue by Khaki01** [swtv-kaist/cs458-fall22#43]
I wanted to clarify.
The final exam is offline in the same classroom, right?

**Comment by moonzoo** [comment]
Yes.  The final exam is offline in the same classroom on Dec 15  (from 1:00 PM to 3:40 PM).
